<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_Courier</class>
 <widget class="QMainWindow" name="MainWindow_Courier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>16</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="title">
     <string>可派送的快递</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delivery">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>800</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>配送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reject">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>800</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>拒绝</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
